views/email/email_partials/breach_alert.hbs

```diff
@@ -1,6 +1,6 @@
     <tr>
       <td valign="top" style="padding-top: 3px; padding-bottom: 10px; width: 70px;">
-        <img src="{{ SERVER_URL }}/img/email_images/missing-logo-icon.png" width="50" style="width: 50px;" />
+        <img src="{{checkImageSrc this.Name}}" alt="" width="50" style="width: 50px"; />
```

Comment:
Is `alt` left intentionally blank? Or should it be something like `alt="{{ this.Name }} logo icon"`?